webapp/lib/charts.dart

```diff
@@ -293,3 +293,115 @@ class SystemEventsTimeseriesLineChartView {
 
   String _stringToHexColor(str) => '#${md5.convert(utf8.encode(str)).toString().substring(0, 6)}';
 }
+
+class DriverTimeseriesBarChartView {
+  DivElement chartContainer;
+  DivElement title;
+  CanvasElement canvas;
+  chartjs.Chart chart;
+  chartjs.ChartData chartData;
+
+  DriverTimeseriesBarChartView() {
+    chartContainer = new DivElement()
+      ..classes.add('chart');
+
+    canvas = new CanvasElement(height: 300, width: 800);
+
+    // Wrap the canvas into a <div> element as needed by Chart.js
+    chartContainer.append(new DivElement()
+      ..classes.add('chart--3080')
+      ..append(canvas));
+
+    title = new DivElement()
+      ..classes.add('chart__title');
+    chartContainer.append(title);
+  }
+
+  void createEmptyChart({String titleText = '', List<String> datasetLabels = const []}) {
+    title.text = titleText;
+
+    List<chartjs.ChartDataSets> chartDatasets = [];
+    datasetLabels.forEach((datasetLabel) {
+      chartDatasets.add(new chartjs.ChartDataSets(
+          label: datasetLabel,
+          backgroundColor: 'rgba(36, 171, 184, 0.3)',
+          borderColor: '#2B8991',
+          data: [],
+          showLine: false,
+          pointRadius: 8));
+      });
+
+    chartData = new chartjs.ChartData(labels: [], datasets: chartDatasets);
+
+    var chartOptions = new chartjs.ChartOptions(
+      legend: new chartjs.ChartLegendOptions(display: false),
+      scales: new chartjs.LinearScale(
+        xAxes: [
+          new chartjs.ChartXAxe()
+            ..type = 'time'
+            ..distribution = 'linear'
+            ..bounds = 'ticks'
+            ..time = (new chartjs.TimeScale(unit: 'day'))
+            ..gridLines = (new chartjs.GridLineOptions(zeroLineWidth: 0))
+        ],
+        yAxes: [
+          new chartjs.ChartYAxe()
+            ..ticks = (new chartjs.LinearTickOptions()..beginAtZero = true)
+            ..gridLines = (new chartjs.GridLineOptions(zeroLineWidth: 0))
+        ]),
+      hover: new chartjs.ChartHoverOptions()..animationDuration = 0,
+      tooltips: new chartjs.ChartTooltipOptions()..mode = 'x-axis',
```

Comment:
An exception is being thrown here 
```
packages/nook_watcher/charts.dart:353:7: Error: No named parameter with the name 'tooltips'.
tooltips: new chartjs.ChartTooltipOptions()..mode = 'x-axis',
```

Reply:
I needed to do an update to the version of `chartjs` package that we're using to support `tooltips`. I'll try to generate a new release for it and update the pubspec, but if you run `pub upgrade` and then the usual `webdev serve` it shouldn't throw the error anymore.

Reply:
Ok...that works